pkg/resources/kafka/kafka.go

```diff
@@ -434,19 +434,19 @@ func (r *Reconciler) reconcileKafkaPodDelete(log logr.Logger) error {
 
 	if len(podsDeletedFromSpec) > 0 {
 		if !arePodsAlreadyDeleted(podsDeletedFromSpec, log) {
-			cruiseControlURL := scale.CruiseControlURLFromKafkaCluster(r.KafkaCluster)
+			cruiseControlURL := cruisecontrol.CruiseControlURLFromKafkaCluster(r.KafkaCluster)
```

Comment:
any specific reason for the package renaming? I guess it makes more sense since we are dealing with CC

Reply:
I expanded the scope of the previous package to avoid duplication since it had some utility functions that were needed for the healer as well.